MANIFEST.in

```diff
@@ -6,7 +6,7 @@ graft icons
 graft scripts/pylint_checkers
 graft doc/img
 graft misc
-include qutebrowser/test/testfile
+include qutebrowser/utils/testfile

```

Comment:
To be perfectly honest I am not a big fan of having static test files in production code.

Shouldn't we rather have a folder inside ``tests`` for static test files?